src/components/Auth/SignUp/SignUp.react.js

```diff
@@ -65,276 +101,200 @@ export default class SignUp extends Component {
     }
   };
 
-  // Handle closing the dialog
-  handleClose = () => {
-    let state = this.state;
-    if (state.success) {
+  closeDialog = () => {
+    const { success } = this.state;
+    if (success) {
       this.setState({
-        msgOpen: false,
+        isDialogOpen: false,
       });
       this.props.onRequestClose();
     } else {
       this.setState({
         email: '',
-        isEmail: false,
-        emailError: true,
-        passwordError: true,
-        passwordConfirmError: true,
-        isCaptchaVerified: false,
-        passwordValue: '',
+        emailError: false,
+        emailErrorMessage: '',
+        password: '',
+        passwordError: false,
+        passwordErrorMessage: '',
+        confirmPassword: '',
+        passwordConfirmError: false,
+        passwordConfirmErrorMessage: '',
         passwordStrength: '',
         passwordScore: -1,
-        confirmPasswordValue: '',
-        msg: '',
+        signupErrorMessage: '',
         success: false,
-        validForm: false,
         serverUrl: '',
         checked: false,
         serverFieldError: false,
-        open: false,
-        msgOpen: false,
+        isDialogOpen: false,
       });
     }
   };
 
-  // Handle toggle between custom server and default server
   handleServeChange = event => {
```

Comment:
Remove the function. We do not have any option for custom server now. Remove related code for the Custom Server implementation